### DISCUSSION TITLE
Deploy mockoon json files to gcp cloud run using github actions

### DISCUSSION THREAD

**Discussion by KalaiJay** [mockoon/mockoon#1208] [Help]
Hi, 
I need to deploy mockoon files to GCP cloud run service. First thing I created a cloud run service in GCP. Then created a repo in visual studio which  contains only the mockoon json file. I need to dockerize, build and then deploy the json files to GCP. Can you please suggest some idea on this?

**Comment by 255kb** [comment]
Hi, 

You will find our Docker image documentation here: https://github.com/mockoon/mockoon/tree/main/packages/cli#docker-image
